### ISSUE TITLE
There is no default value for opts in createAdapter

### ISSUE THREAD

**Issue by iMokhonko** [socketio/socket.io-aws-sqs-adapter#2]
So I was trying to use `createAdapter` function based on [documentation example](https://socket.io/docs/v4/aws-sqs-adapter/#usage) and it errored out with this error.

```
const topicName = opts.topicName || "socket-io";
                           ^
TypeError: Cannot read properties of undefined (reading 'topicName')
```
Basically if you do not provide options object it will be undefined. Maybe it will be better to rewrite like this

```
const topicName = opts?.topicName ?? "socket-io";
```

**Comment by darrachequesne** [comment]
Thanks for the heads-up! There should indeed be a default value :+1:

**Comment by darrachequesne** [comment]
This should be fixed by https://github.com/socketio/socket.io-aws-sqs-adapter/commit/cda506606e4fc4d41e7e71db2b41c81655cc6aa2, included in version `0.1.1`. Thanks for the heads-up!
